src/pages/MoviePost/MoviePost.jsx

```diff
@@ -1,7 +1,220 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { LOCAL_URL } from '../../config';
+import Posts from './Posts';
+import PostModal from './PostModal';
+import backimg from '../../image/sdf.jpeg';
+import AllRankMovies from './AllRankMovies';
+import styled from 'styled-components';
+
+const LIMIT = 4;
 
 function MoviePost() {
-  return <div>MoviePost</div>;
+  const [rankMovies, setRankMoives] = useState([]);
+  const [movieList, setMovieList] = useState([]);
+  const [modalPost, setModalPost] = useState([]);
+  const [showPostModal, setShowPostModal] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+  const [pagination, setPagination] = useState({
+    offset: 0,
+    limit: LIMIT,
+    totalPage: 0,
+    currentPage: 1,
+  });
+  const history = useHistory();
+
+  const onMoreClick = () => {
+    setPagination({
+      ...pagination,
+      currentPage: pagination.currentPage + 1,
+      offset: pagination.offset + pagination.limit,
+    });
+  };
+
+  const onMoveWrite = () => {
```

Comment:
```suggestion
  const goToWritePage = () => {
```
이렇게 이름 바꾸는 건 어떨까요? `on-`이 붙으면 왠지 이벤트 리스너 같아서요

Reply:
넵! 감사합니다